pages/index.haml

```diff
@@ -17,8 +17,144 @@
+          are united behind a fair contract. Your presence helps ensure our voices are heard.
+
+    %aside
+      %h4 Top Demands We’re Rallying For
+
+      %ul
+        %li A company-wide minimum salary.
+        %li Protections against AI and contractor replacement of our roles.
+        %li Codifying the four-day (32-hour) workweek we've worked under for more than three years.
+
+    / Structured data for better event previews
+    %script{ type: "application/ld+json" }
+      :plain
+        {
+          "@context": "https://schema.org",
+          "@type": "Event",
+          "name": "Kickstarter United - Solidarity Rally",
+          "startDate": "2025-09-05T17:00:00-04:00",
+          "endDate": "2025-09-05T18:00:00-04:00",
+          "eventAttendanceMode": "https://schema.org/OnlineEventAttendanceMode",
+          "eventStatus": "https://schema.org/EventScheduled",
+          "description": "Kickstarter United is rallying support for a fair contract: a codified four-day, 32-hour workweek, a company-wide minimum salary, and protections against replacement by contractors or AI.",
+          "organizer": {
+            "@type": "Organization",
+            "name": "Kickstarter United (OPEIU Local 153)"
+          },
+          "location": {
+            "@type": "VirtualLocation",
+            "url": "#{rally_registration_url}"
+          },
+          "offers": {
+            "@type": "Offer",
+            "url": "#{rally_registration_url}",
+            "price": "0",
+            "priceCurrency": "USD",
+            "availability": "https://schema.org/InStock"
+          },
+          "performer": [
+            {
+              "@type": "Organization",
+              "name": "The New York Times Tech Guild",
+              "member": [
+                { "@type": "Person", "name": "John Cruickshank" },
+                { "@type": "Person", "name": "Kait Hoehne" }
+              ]
+            },
+            {
+              "@type": "Organization",
+              "name": "Alphabet Workers Union",
+              "member": [
+                { "@type": "Person", "name": "Parul Koul", "jobTitle": "President" }
+              ]
+            },
+            {
+              "@type": "Person",
+              "name": "Emily Gallagher",
+              "jobTitle": "New York State Assemblymember"
+            },
+            {
+              "@type": "Person",
+              "name": "Phara Souffrant Forrest",
+              "jobTitle": "New York State Assemblymember"
+            },
+            {
+              "@type": "Person",
+              "name": "Kshama Sawant",
+              "jobTitle": "Former Seattle City Councilmember and candidate for the United States Congress"
+            }
+          ],
+          "image": [
+            "https://kickstarterunited.org/images/logo.svg"
+          ]
+        }
+
+  %h4 Sign Our Solidarity Petition
```

Comment:
lol I'm annoying but what about some sort of divider/ border between the rally and petition